FreeRTOS/Source/portable/GCC/RISC-V/port.c

```diff
@@ -196,7 +196,8 @@ void *pvReturnSealer;
 		               __CHERI_CAP_PERMISSION_PERMIT_LOAD_CAPABILITY__);
 
 	pvReturnSealer = cheri_setaddress( pvAlmightyDataCap, SANDBOX_RETURN_OTYPE );
-	pxPortSandboxReturnFunc = cheri_seal( xPortSandboxReturn, pvReturnSealer );
+	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, xPortSandboxReturn );

```

Comment:
Does this not need a cast to integer?
```suggestion
	pxPortSandboxReturnFunc = cheri_setaddress( pvAlmightyCodeCap, (ptraddr_t)xPortSandboxReturn );

```